submission/proxy.go

```diff
@@ -39,6 +39,16 @@ const (
 	AppleCTPolicy
 )
 
+var (
+	proxyOnce    sync.Once
+	llUpdCounter monitoring.Counter
```

Comment:
ll ? 11 ?  what's this?

Reply:
maybe logListUpdates  and drop the word 'Counter'.  It's explicit where you create it that it's a Counter, and later logListUpdates.Inc() makes implicit that it's a Counter too.

Reply:
Renamed per suggestion.